applications/nrf_desktop/src/hw_interface/Kconfig.battery_meas

```diff
@@ -22,6 +22,13 @@ config DESKTOP_BATTERY_MEAS
 
 endchoice
 
+config DESKTOP_BATTERY_MEAS_DEF_PATH
+	string "Configuration file"
```

Comment:
What is this file configuring ?

It's describing the characteristics of the battery as far as I can tell, but it's not configuring anything.

Reply:
Characteristic of the battery (mapping between measured voltage and state of charge) is "configurable" by user depending on battery that is used in hardware.